### PR TITLE
Update description for native contract

### PR DESCRIPTION
**Pull request by leeyr338**
更改原生合约的描述。

### CODE REVIEW

**Review by jerry-yu — DISMISSED** [review]

File: docs/cita/advanced-use/contracts/rust.md [hunk 1]
```diff
@@ -263,17 +278,112 @@ rpc getTransactionReceipt --hash 0x9c6bae3216bbaa755f80e61d00cd3502e151a61bafca6
...
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": "0x0000000000000000000000000000000000000000000000000000000000000011"
+}
 ```
 
 符合我们编写的智能合约预期。
 
+### 附：原生合约 hello.rs 的完整代码
```
Comment (kaikai1024) [inline docs/cita/advanced-use/contracts/rust.md]: Maybe you can put the code here:

https://github.com/cryptape/test-contracts/blob/master/rust_hello.rs

Comment (leeyr338) [reply docs/cita/advanced-use/contracts/rust.md]: That sounds better!

Comment (leeyr338) [reply docs/cita/advanced-use/contracts/rust.md]: Pls merge this PR first.
https://github.com/cryptape/test-contracts/pull/8

Comment (kaikai1024) [reply docs/cita/advanced-use/contracts/rust.md]: done

Comment (leeyr338) [reply docs/cita/advanced-use/contracts/rust.md]: The source code has referenced to https://github.com/cryptape/test-contracts/blob/master/hello.rs .

**Review by kaikai1024 — COMMENTED** [review]
And the link:

2019-10-12-101819_652x73_scrot

**Comment by leeyr338** [comment]
> And the link:
> 
> 2019-10-12-101819_652x73_scrot

Yes, I missed it!

**Review by kaikai1024 — APPROVED** [review]

**Review by jerry-yu — APPROVED** [review]

**Pull request merged**